### ISSUE TITLE
keepasshttp incompatible

### ISSUE THREAD

**Issue by JadelDXB** [pfn/passifox#378]
Hi, 

Everything was working ok for a couple of weeks until today where it says plugin incompatible when I launch keepass.

Windows 8.1 keepass 2.29 (and 2.28. and 2.27)

I have tried,

uninstall keepass, delete plugin and plugin cache, then reinstall all. 
Used manual install and chocolatey

Install old version keepasshttp 
install old version keepass.

update netframework 4.5, disable/reenable netframework 3.5/2.0 compatibilty in control panel
Same result

Frustrated

Thanks

**Comment by bruev** [comment]
Hi,

I have the same problem.

**Comment by TheFragen** [comment]
Running Windows 8.1 64 bit
Keepass 2.29
Newest KeePassHTTP

The plugin doesn't even show up in the new version of Keepass.

Measures tried so far:
1. Uninstalling KeePassHTTP, restarting, installing KeePassHTTP again.
2. Uninstalling KeePass and deleting all leftovers, uninstalling KeePassHTTP. Then installing everything again.

The only fix is to revert to KeePass 2.28.

**Comment by erhard895** [comment]
Hi,

I have the same problem, KeePass was working ok for years until today where it says:  
"Folgendes Plugin ist inkompatibel mit der aktuellen KeePass-Version:
C:\Users\.....\AppData\Local\KeePass\PluginCach\KeePassHttp.dll"

Running Windows 8.1 64 bit
Keepass 2.29
Newest KeePassHTTP

I have tried:
- uninstall keepass, delete plugin and plugin cache, then reinstall all.
- Used manual install and chocolatey
- Checking my system with sfc and dism - no errors found

But I get every time the same massage.

Thank you for help

**Comment by zanderfray** [comment]
Same here, stopped working to today, tried all the same things.

**Comment by erhard895** [comment]
Hi,

KeePass is working properly after I installed the last update.

**Comment by zanderfray** [comment]
I have solved for me, it was a problem with astrill vpn software, after uninstalling astrill the problem cleared.  

After further testing I have found that Astrill can be installed without causing a problem, it is only after I have used it in firefox that the error starts, which is why it appeared to start at a random time...

**Comment by a-m-s** [comment]
I had the exact same problem.

It turned out that there was a half-dead mono process still running in the background. It was holding the keepasshttp port open (19455) and thus preventing a new instance from starting. Killing the process fixed the problem. Of course, a reboot would have worked also.

Anyway, this is not a passifox/chromeipass bug.

**Comment by JadelDXB** [comment]
Are you seriously suggesting that the solution is restarting the pc. Wow you are a genius! 

Of course that's not it.

**Comment by a-m-s** [comment]
No, I'm suggesting you check "netstat -nltp" and see if anything has port 19455 open.

Actually, after I'd spent two hours trying to figure what the hell had changed, and then finally figured it out, I kinda wished I had just tried a reboot first. I would have saved a lot stress!

I'm not suggesting that your problem is definitely the same as mine; only that others landing here might have the same problem I did. The symptoms are identical.
